test/sharness/t0180-private-network.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+#
+# Copyright (c) 2015 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="Test private network feature"
+
+. lib/test-lib.sh
+
+test_init_ipfs
+
+export LIBP2P_FORCE_PNET=1
```

Comment:
Agreed with this, but i'm wanting to avoid using config values for features that are currently experimental. Once private networks are more well defined, we can make it an official feature and give it config values to use